src/components/notifications/static/__snapshots__/static.spec.js.snap

```diff
@@ -0,0 +1,93 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
We're trying to get rid of `enzyme` and snapshot tests in ui-kit. Could you remove this one?